### PR TITLE
Revamped .NET MAUI Tools controls Getting Started UG

### PR DESCRIPTION
**Pull request by Kamalesh-Periyasamy**
Tools controls:
1. Button
2. Chips
3. Slider
4. RangeSelector
5. DateTime Range Selector

### CODE REVIEW

File: MAUI/Button/Getting-Started.md [hunk 1]
```diff
@@ -8,24 +8,37 @@ documentation: ug
 ---
 
 # Getting Started with .NET MAUI Button
+This section guides you through setting up and configuring a [`Button`](https://www.syncfusion.com/maui-controls/maui-button) in your .NET MAUI application. Follow the steps below to add a basic Button to your project.
```
Comment (PrithisIyyappan) [inline MAUI/Button/Getting-Started.md:11]: Remove the highlights.

File: MAUI/Button/Getting-Started.md [hunk 2]
```diff
@@ -8,24 +8,37 @@ documentation: ug
...
-## Adding a .NET MAUI Button reference
+## Step 1: Create a New MAUI Project
+
+1. Open Visual Studio or VS Code.
+2. Go to **File > New > Project** and choose the **.NET MAUI App** template.
+3. Name the project and choose a location, then click **Create**.
+
+## Step 2: Install the Syncfusion MAUI Buttons NuGet Package
+
+1. In **Solution Explorer**, right-click the project and choose Manage NuGet Packages.
```
Comment (PrithisIyyappan) [inline MAUI/Button/Getting-Started.md:33]: Bold Manage Nuget Packages

File: MAUI/Button/Getting-Started.md [hunk 3]
```diff
@@ -8,24 +8,37 @@ documentation: ug
...
-The Syncfusion .NET MAUI controls are available in [Nuget.org](https://www.nuget.org/). To add [`.NET MAUI Button`](https://www.syncfusion.com/maui-controls/maui-button) to your project, open the NuGet package manager in Visual Studio, search for [`Syncfusion.Maui.Buttons`](https://www.nuget.org/packages/Syncfusion.Maui.Buttons), and install it.
 
-## Handler registration 
+## Step 3: Register the Handler 
 
-In the MauiProgram.cs file, register the handler for the Syncfusion core.
+Syncfusion.Maui.Core Nuget is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.
 
 {% highlight c# hl_lines="6 17" %}
 using Microsoft.Maui;
```
Comment (PrithisIyyappan) [inline MAUI/Button/Getting-Started.md:44]: Where are other steps?

File: MAUI/Button/Getting-Started.md [hunk 4]
```diff
@@ -8,24 +8,44 @@ documentation: ug
...
 
-{% youtube "https://www.youtube.com/watch?v=KTGkYShi1YE" %}
+Install [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later.
+
+Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as described [here](https://github.com/dotnet/maui/wiki/VS-Code-setup).
+
+## Step 1: Create a New MAUI Project
 
-## Adding a .NET MAUI Button reference
+Visual Studio
```
Comment (Choza-rajan) [inline MAUI/Button/Getting-Started.md:27]: The title 3 hash tag missed. See [this ](https://github.com/syncfusion-content/maui-docs/pull/2643/files?diff=split&w=1)PR for more info

File: MAUI/Button/Getting-Started.md [hunk 5]
```diff
@@ -8,24 +8,44 @@ documentation: ug
...
-In the MauiProgram.cs file, register the handler for the Syncfusion core.
+1. In **Solution Explorer**, right-click the project and choose **Manage NuGet Packages**.
+2. Search for [Syncfusion.Maui.Buttons](https://www.nuget.org/packages/Syncfusion.Maui.Buttons) and install the latest version.
+3. Ensure the necessary dependencies are installed correctly, and the project is restored.
+
+
+
+## Step 3: Register the Handler 
+
+[Syncfusion.Maui.Core nuget](https://www.nuget.org/packages/Syncfusion.Maui.Core) is a dependent package for all Syncfusion controls of .NET MAUI. In the **MauiProgram.cs file**, register the handler for Syncfusion core.
```
Comment (Choza-rajan) [inline MAUI/Button/Getting-Started.md:48]: why nuget is added inside the link?

File: MAUI/Button/Getting-Started.md [hunk 6]
```diff
@@ -98,7 +118,7 @@ SfButton button = new SfButton();
 
 ## Button icon
 
-The button icon can be defined using the [`ImageSource`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.ButtonBase.html#Syncfusion_Maui_Core_ButtonBase_ImageSource) and [`ShowIcon`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.ButtonBase.html#Syncfusion_Maui_Core_ButtonBase_ShowIcon) properties of [`SfButton`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfButton.html).
+The button icon can be defined using the [ImageSource](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.ButtonBase.html#Syncfusion_Maui_Core_ButtonBase_ImageSource) and [ShowIcon](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.ButtonBase.html#Syncfusion_Maui_Core_ButtonBase_ShowIcon) properties of [SfButton](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfButton.html).
```
Comment (Choza-rajan) [inline MAUI/Button/Getting-Started.md:121]: Add the image **note** section before the code snippet

File: MAUI/Chips/Getting-Started.md [hunk 7]
```diff
@@ -8,16 +8,42 @@ documentation: ug
...
+Before proceeding, ensure the following are in place:
 
-In the MauiProgram.cs file, register the handler for Syncfusion core.
+Install [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later.
+
+Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as described [here](https://github.com/dotnet/maui/wiki/VS-Code-setup).
+
+## Step 1: Create a New MAUI Project
+
+Visual Studio
```
Comment (Choza-rajan) [inline MAUI/Chips/Getting-Started.md:27]: recheck and add the header tag

File: MAUI/Chips/Getting-Started.md [hunk 8]
```diff
@@ -8,16 +8,42 @@ documentation: ug
...
+Visual Studio Code
+1. Open the command palette by pressing `Ctrl+Shift+P`` and type **.NET:New Project** and enter.
+2. Choose the **.NET MAUI App** template.
+3. Select the project location, type the project name and press enter.
+Then choose **Create project**
+
+## Step 2: Install the Syncfusion MAUI Buttons NuGet Package
+
+1. In **Solution Explorer**, right-click the project and choose **Manage NuGet Packages**.
+2. Search for [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core) and install the latest version.
```
Comment (Choza-rajan) [inline MAUI/Chips/Getting-Started.md:41]: buttons nuget is not explained here

File: MAUI/Chips/Getting-Started.md [hunk 9]
```diff
@@ -8,16 +8,42 @@ documentation: ug
...
+3. Ensure the necessary dependencies are installed correctly, and the project is restored.
+
+## Step 3: Register the Handler 
+
+[Syncfusion.Maui.Core nuget](https://www.nuget.org/packages/Syncfusion.Maui.Core) is a dependent package for all Syncfusion controls of .NET MAUI. In the **MauiProgram.cs file**, register the handler for Syncfusion core.
```
Comment (Choza-rajan) [inline MAUI/Chips/Getting-Started.md:46]: remove the nuget from link

File: MAUI/DateTime-Range-Selector/getting-started.md [hunk 10]
```diff
@@ -8,29 +8,43 @@ documentation: ug
 ---
 
 # Getting Started with .NET MAUI DateTime Range Selector
-
-This section explains the steps required to add the DateTime Range Selector control and its elements such as track, ticks, labels and tooltip. This section covers only the basic features needed to know to get started with the Syncfusion DateTime Range Selector.
+This section guides you through setting up and configuring a [DateTime Range Selector](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Sliders.SfDateTimeRangeSelector.html) in your .NET MAUI application. Follow the steps below to add a basic Button to your project.
```
Comment (Choza-rajan) [inline MAUI/DateTime-Range-Selector/getting-started.md:11]: basic Button to your project -- mentioned button as control name in here

File: MAUI/Range-Selector/getting-started.md [hunk 11]
```diff
@@ -6,30 +6,44 @@ platform: maui
 control: SfRangeSelector
 documentation: ug
 ---
-
 # Getting Started with .NET MAUI Range Selector
-
-This section explains the steps required to add the Range Selector control and its elements such as track, ticks, labels and tooltip. This section covers only basic features needed to know to get started with Syncfusion Range Selector.
+This section guides you through setting up and configuring a [Range Selector](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Sliders.SfRangeSelector.html) in your .NET MAUI application. Follow the steps below to add a basic Button to your project.
```
Comment (Choza-rajan) [inline MAUI/Range-Selector/getting-started.md:10]: basic Button to your project -> used button as a control name

File: MAUI/Slider/getting-started.md [hunk 12]
```diff
@@ -6,32 +6,44 @@ platform: maui
 control: SfSlider
 documentation: ug
 ---
-
 # Getting Started with .NET MAUI Slider
-
-This section explains the steps required to add the slider control and its elements such as track, ticks, labels and tooltip. This section covers only basic features needed to know to get started with Syncfusion Slider.
+This section guides you through setting up and configuring a [Slider](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Sliders.SfSlider.html) in your .NET MAUI application. Follow the steps below to add a basic Chip to your project.
```
Comment (Choza-rajan) [inline MAUI/Slider/getting-started.md:10]: basic Chip to your project --> used chip is control name

File: MAUI/Slider/getting-started.md [hunk 13]
```diff
@@ -6,32 +6,44 @@ platform: maui
...
 
- The Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add the SfSlider to your project, open the NuGet package manager in the Visual Studio, search for the Syncfusion.Maui.Sliders, and install it.
+Visual Studio Code
+1. Open the command palette by pressing `Ctrl+Shift+P`` and type **.NET:New Project** and enter.
+2. Choose the **.NET MAUI App** template.
+3. Select the project location, type the project name and press enter.
+Then choose **Create project**
 
- ![Create MAUI Application](images/getting-started/nuget-installation.png)
+## Step 2: Install the Syncfusion MAUI Buttons NuGet Package
```
Comment (Choza-rajan) [inline MAUI/Slider/getting-started.md:38]: used button name

File: MAUI/Slider/getting-started.md [hunk 14]
```diff
@@ -6,32 +6,44 @@ platform: maui
...
 
+1. In **Solution Explorer**, right-click the project and choose **Manage NuGet Packages**.
+2. Search for [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core) and install the latest version.
```
Comment (Choza-rajan) [inline MAUI/Slider/getting-started.md:41]: missed the details of slider nuget

**Review by Choza-rajan — APPROVED** [review]

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**